DgmlPowerTools.2022/DgmlPowerTools.2022/Utilities/ProjectDependencies.cs

```diff
@@ -227,25 +225,36 @@ private static void CreateNugetReferences(string sourceProjectFullName, Graph gr
                     string name = (string)e.Attribute("Include");
                     if (!string.IsNullOrEmpty(name))
                     {
-                        string version = (string)e.Attribute(ns + "Version");
-                        if (string.IsNullOrEmpty(version))
+                        string nugetVersionAttribute = (string)e.Attribute(ns + "Version");
+                        if (string.IsNullOrEmpty(nugetVersionAttribute))
                         {
-                            version = (string)e.Element(ns + "Version");
+                            nugetVersionAttribute = (string)e.Element(ns + "Version");
                         }
                         string id = name;
                         string label = name;
-                        Version v = null;
-                        if (!string.IsNullOrEmpty(version) && Version.TryParse(version, out v))
+                        Version nugetVersion = null;
+                        if (!string.IsNullOrEmpty(nugetVersionAttribute) && Version.TryParse(nugetVersionAttribute, out nugetVersion))
```

Comment:
do we really even need to parse the "Version" object?  Or can we just deal with the string value?

Reply:
The parsing came actually from you with your last commit. I dealt before already with just the string value 😃 
See: https://github.com/clovett/DgmlPowerTools/commit/10444b6224321a07fd4960bf02bec1c31f12bf2f

Reply:
I'm pretty sure you had "new Version(str)" also.

Reply:
Apology. I thought you meant the dedicated `TryParse` statement. 
Changed it to string.

Reply:
Cool thanks.